src/csharp/Grpc.Core/Server.cs

```diff
@@ -141,7 +141,9 @@ public int RequestCallTokensPerCompletionQueue
 
         /// <summary>
         /// Starts the server.
-        /// Throws <c>IOException</c> if not successful.
+        /// Throws <c>IOException</c> if not all ports have been bound successfully (see <c>Ports.Add</c> method).
```

Comment:
Minor: there's [`<see cref="..." />`](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/see) for embedding links to other methods/properties. Not sure what the style in this code base is.